torch/csrc/jit/dynamic_dag.h

```diff
@@ -187,7 +188,7 @@ size_t DynamicDAG<T>::debugNumVertices() const {
 
 template <typename T>
 Vertex<T>* DynamicDAG<T>::newVertex(T datum) {
-  vertices_.emplace_back(new Vertex<T>(vertices_.size(), datum));
+  vertices_.push_back(torch::make_unique<Vertex<T>>(vertices_.size(), datum));
```

Comment:
What was wrong with that code...

Reply:
It uses `new`?

Reply:
On a single argument, which is perfectly safe. Actually the code faster was even potentially faster, because you were previously constructing the pointer in-place, while now you're constructing a copy on the stack, and then calling a move constructor.